angularjs-portal-mock-portal/src/main/webapp/api/layoutDoc

```diff
@@ -172,7 +172,9 @@
          "widgetURL":null,
          "widgetType":null,
          "widgetTemplate":null,
-         "widgetConfig":null,
+         "widgetConfig": {
+            "launchText" : "Free Your Mind"

```

Comment:
👍  Really appreciate this. Quality mock data helps with exercising, demonstrating, staving off regressions in the product.